### PR TITLE
style: allow code block to be copied using the copy button

### PR DESCRIPTION
**Pull request by mcanouil**
This PR tweaks the code blocks in tables to allow the copy button which make it easier to use the code showed.
It also makes the OS order consistent across the website.

| Before | After |
|--------|-------|
| image | image |

### CODE REVIEW

**Review by cderv — APPROVED** [review]
Thanks a lot ! This looks nice. 

The `filename` with Termainl does not add much for those code blocks inside tables, it takes some space in the table. However, I understand this is to be coherent with other code blocks in the doc. 

@cwickham just added you as reviewer so that you can decide to merge in the doc when ready

**Comment by mcanouil** [comment]
At first I did not add the "Terminal" part.
What motivated this were:
- consistency with other parts of the website
- a previous discussion with user that did not know where to paste the command

**Comment by cderv** [comment]
> a previous discussion with a user that did not know where to paste the command

Oh good user feedback ! I would have issue it was known... Makes sense to add it for that then !

File: docs/computations/_jupyter-install.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ If you are in a fresh Python 3 environment, installing the `jupyter` package wil
 | Pkg. Manager | Command                        |
 +==============+================================+
 | Pip\         | ``` bash                       |
-| (Windows)    | py -m pip install jupyter      |
+| (Mac/Linux)  | python3 -m pip install jupyter |
 |              | ```                            |
 +--------------+--------------------------------+
 | Pip\         | ``` bash                       |
-| (Mac/Linux)  | python3 -m pip install jupyter |
+| (Windows)    | py -m pip install jupyter      |
```
Comment (cwickham) [inline docs/computations/_jupyter-install.md:13]: Is there a reason the code cells in this table don't have `filename="Terminal"`?

Comment (mcanouil) [reply docs/computations/_jupyter-install.md:13]: No, I simply did not "see" it at my second pass when adding it "everywhere".

Comment (mcanouil) [reply docs/computations/_jupyter-install.md]: Now fixed.

**Pull request merged**